pkg/providers/imagefamily/customscriptsbootstrap/provisionclientbootstrap.go

```diff
@@ -80,6 +80,10 @@ func (p ProvisionClientBootstrap) GetCustomDataAndCSE(ctx context.Context) (stri
 	labels := lo.Assign(map[string]string{}, p.Labels)
 	getAgentbakerGeneratedLabels(p.ResourceGroup, labels)
 
+	// artifact streaming is not yet supported for Arm64, for Ubuntu 20.04, and for Azure Linux v3
```

Comment:
Can the Ubuntu 20.04 check be added too?

Reply:
we dont support ubuntu 20.04

Reply:
Cool, do you plan to support it in the future?

Reply:
maybe if fips + cvm in SIG aren't in 2404 or 2204